custom/icds_reports/utils/fix_child_health_missing_month.sql

```diff
@@ -0,0 +1,305 @@
+        wasting_severe_v2 = ut.wasting_severe_v2,
+        zscore_grading_hfa_normal = ut.zscore_grading_hfa_normal,
+				zscore_grading_hfa_moderate = ut.zscore_grading_hfa_moderate,
+				zscore_grading_hfa_severe = ut.zscore_grading_hfa_severe,
+				zscore_grading_hfa_recorded_in_month = ut.zscore_grading_hfa_recorded_in_month,
+				zscore_grading_wfh_recorded_in_month = ut.zscore_grading_wfh_recorded_in_month
+    from (
+        SELECT * from temp_agg_Child_my
+    ) ut 
+    where (
+        agg_child_health.awc_id=ut.awc_id and 
+        agg_child_health.month=ut.month and
+        agg_child_health.gender=ut.sex and
+        agg_child_health.age_tranche=ut.age_tranche and
+        agg_child_health.caste=ut.caste and
+        agg_child_health.disabled=ut.coalesce_disabled and
+        agg_child_health.minority = ut.coalesce_minority and
+        agg_child_health.resident = ut.coalesce_resident
+            );
+
+Roll ups:
+
+UPDATE "agg_child_health_2018-05-01_4" agg_child_health
+    SET wasting_moderate = ut.wasting_moderate,
+        wasting_severe = ut.wasting_severe,
+        stunting_moderate = ut.stunting_severe,
+        stunting_severe = ut.stunting_severe,
+        wasting_normal = ut.wasting_normal,
+        stunting_normal = ut.stunting_normal,
+        wasting_normal_v2 = ut.wasting_normal_v2,
+        wasting_moderate_v2 = ut.wasting_moderate_v2,
+        wasting_severe_v2 = ut.wasting_severe_v2,
+        zscore_grading_hfa_normal = ut.zscore_grading_hfa_normal,
+				zscore_grading_hfa_moderate = ut.zscore_grading_hfa_moderate,
+				zscore_grading_hfa_severe = ut.zscore_grading_hfa_severe,
+				zscore_grading_hfa_recorded_in_month = ut.zscore_grading_hfa_recorded_in_month,
+				zscore_grading_wfh_recorded_in_month = ut.zscore_grading_wfh_recorded_in_month
+    from (
+        SELECT
+        supervisor_id,
+        gender,
+        age_tranche,
+        SUM(wasting_moderate) as wasting_moderate,
+        SUM(wasting_severe) as wasting_severe,
+        SUM(stunting_moderate) as stunting_moderate,
+        SUM(stunting_severe) as stunting_severe,
+        SUM(wasting_normal) as wasting_normal,
+        SUM(stunting_normal) as stunting_normal,
+        SUM(wasting_normal_v2) as wasting_normal_v2,
+        SUM(wasting_moderate_v2) as wasting_moderate_v2,
+        SUM(wasting_severe_v2) as wasting_severe_v2,
+        SUM(zscore_grading_hfa_normal) as zscore_grading_hfa_normal,
+				SUM(zscore_grading_hfa_moderate) as zscore_grading_hfa_moderate,
+				SUM(zscore_grading_hfa_severe) as zscore_grading_hfa_severe,
+				SUM(zscore_grading_hfa_recorded_in_month) as zscore_grading_hfa_recorded_in_month,
+				SUM(zscore_grading_wfh_recorded_in_month) as zscore_grading_wfh_recorded_in_month
+
+        FROM "agg_child_health_2018-05-01_5" agg_child INNER JOIN (SELECT DISTINCT ucr.doc_id FROM "ucr_icds-cas_static-awc_location_88b3f9c3" ucr WHERE ucr.awc_is_test=0) tt ON tt.doc_id = agg_child.awc_id
```

Comment:
you can use "awc_location_local" table instead of "ucr_icds-cas_static-awc_location_88b3f9c3" with where clause of aggregation_level=5. AS "ucr_icds-cas_static-awc_location_88b3f9c3" table is no more used on dashboard.

Reply:
same for all queries below.